### ISSUE TITLE
2024 DevOps Community of Practice Agendas

### ISSUE THREAD

**Issue by chelseybeck** [hackforla/ops#100]
### Overview
This issue tracks meeting agendas and summaries for the OPS COP meetings.

#### Template for agenda
```
## 2024-MM-DD
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)
```
### Agenda Quick-Links
[30 Oct](https://github.com/hackforla/ops/issues/100#issuecomment-2448778631)
[23 Oct](https://github.com/hackforla/ops/issues/100#issuecomment-2418323333)
[16 Oct](https://github.com/hackforla/ops/issues/100#issuecomment-2418213230)
[09 Oct](https://github.com/hackforla/ops/issues/100#issuecomment-2380219001)
[18 Sep](https://github.com/hackforla/ops/issues/100#issuecomment-2353736236)
[11 Sep](https://github.com/hackforla/ops/issues/100#issuecomment-2327882245)
[28 Aug](https://github.com/hackforla/ops/issues/100#issuecomment-2313658142)
[21 Aug](https://github.com/hackforla/ops/issues/100#issuecomment-2294977442)
[14 Aug](https://github.com/hackforla/ops/issues/100#issuecomment-2274815187)
[29 May](https://github.com/hackforla/ops/issues/100#issuecomment-2138090836)
[22 May](https://github.com/hackforla/ops/issues/100#issuecomment-2115533879)
[15 May](https://github.com/hackforla/ops/issues/100#issuecomment-2111495125)
[10 April](https://github.com/hackforla/ops/issues/100#issuecomment-2048602202)
[27 March](https://github.com/hackforla/ops/issues/100#issuecomment-2024175614)
[13 March](https://github.com/hackforla/ops/issues/100#issuecomment-1987105428)
[28 Feb](https://github.com/hackforla/ops/issues/100#issuecomment-1970099096)
[21 Feb](https://github.com/hackforla/ops/issues/100#issuecomment-1954959920)
[14 Feb](https://github.com/hackforla/ops/issues/100#issuecomment-1920452541)
[31 Jan](https://github.com/hackforla/ops/issues/100#issuecomment-1915702723)
[24 Jan](https://github.com/hackforla/ops/issues/100#issuecomment-1906942997)
[17 Jan](https://github.com/hackforla/ops/issues/100#issuecomment-1906920204)
[10 Jan](https://github.com/hackforla/ops/issues/100#issuecomment-1906919662)

---

[2022/2023 Agendas](https://github.com/hackforla/ops/issues/12)

---

### Status for the new year
- Joshua Gagnier - new
- Kazushi Rickert - has created a new issue https://github.com/hackforla/ops/issues/94
- Shikha - new (interested in incubator)
- Paul Espinosa - 
   - Working with Tyler on Home Unite Us incubator .  Tyler has submitted PR to ops in October PR https://github.com/hackforla/incubator/pull/26
   - How is AWS organized and how HUU fits into it.
- Tyson
   - Working on creating a GHA to help review PRs (it will show what was created and destroyed) and its documentation 
- Sidharth - returning
- Ernie 
   - Civic Tech Index https://github.com/civictechindex/CTI-website-backend/issues/100
- Robin
  -  Working with Bonnie and Jason to sort out the issues between incubator and ops.

**Comment by chelseybeck** [comment]
## 10 January
_[Back to Top](#)_

### AGENDA


#### Notes from meeting


#### Task Items

**Comment by chelseybeck** [comment]
## 17 January
_[Back to Top](#)_

### AGENDA


#### Notes from meeting


#### Task Items

**Comment by chelseybeck** [comment]
## 24 January
_[Back to Top](#)_

### AGENDA
_Timezones listed as PDT_

- [ ] 18:00 _5 minutes_ - Welcome!
   - Rename yourself to [NAME | PROJECT]
   - Post your LinkedIn link in the chat
   - Open Lead positions
      - (2) Co-lead 
- [ ] 18:05 _5 minutes_ - Co-lead/new member intros
- [ ] 18:10 _10 minutes_ - Project support
- [ ] 18:20 _30 minutes_ - AWS Identity and Access Management (IAM) Discussion
  - [ ] IAM groups & associated policies
  - [ ] IAM Identity Center
- [ ] 18:50 _10 minutes_ - Any Other Business (AOB)


#### Notes from meeting


#### Task Items


#### Meeting Attendance

CoP Members
- [ ]

**Comment by ExperimentsInHonesty** [comment]
Create Levels
### Join HfLA and DevOps
1. Fundamentals and Concepts
   1. Learn Terraform
       1. Setup personal AWS
       2. Do HashiCorp Tutorials
   3. Deploy a Terraform hello world project to our GitHub Repo
   4. Get read access to infrastructure organization (see resources)
### Get assigned to project as their DevOps person
3. Administration
    1. get access to do stuff to that projects' deployment
### Become a mentor across all projects DevOps people
5. expert level
   1. Ability to do stuff to any project 
#### Permissions and Resources management
6. IAM supervisor / Services Supervisor (is added to the 
Management Group on AWS)
    1. Dispenses new permissions and removes permissions for user 
    2. Add new services in partnership with Owner
7. Billing (owner) (is added to the Cost Management Group)

**Comment by ExperimentsInHonesty** [comment]
Notes from meeting

- [ ] Create a repo just for IAM permissions.  What is going to be called?
- [ ] We need an issue for research to determine how to attach user to resources of a project.
- [ ] Decide what policies are needed for level 2 and beyond.

**Comment by BrianCodes33** [comment]
Cool Sent from my iPhoneOn Jan 24, 2024, at 6:40 PM, Bonnie Wolfe ***@***.***> wrote:﻿
Notes from meeting

 Create a repo just for IAM permissions.  What is going to be called?
 We need an issue for research to determine how to attach user to resources of a project.
 Decide what policies are needed for level 2 and beyond.

**Comment by chelseybeck** [comment]
## 31 January
_[Back to Top](#)_

### AGENDA
_Timezones listed as PDT_

- [ ] 18:00 _5 minutes_ - Welcome!
   - [ ] Open Lead positions
      - [Co-lead](https://github.com/hackforla/ops/projects/1#column-19747877)
   - [ ] Intros
- [ ] 18:05 _10 minutes_ - Project support
- [ ] 18:15 _30 minutes_ - AWS Identity and Access Management (IAM) Discussion
  - [ ] Decide what policies are needed for level 2 and beyond.
    - [ ] IAM [groups](https://github.com/hackforla/ops/issues/96) & associated [policies](https://github.com/hackforla/ops/issues/95)
      - [IAM Groups & Policies](https://docs.google.com/document/d/1OUwPCzzHa2YROSp7YugYG_LH-NNIiroweqVxxSrQ4gY/edit?usp=sharing)
  - [ ] IAM Identity Center
- [ ] 18:45 _10 minutes_ - Review [board](https://github.com/hackforla/ops/projects/4) and assign new issues

### Resources
[IAM Groups & Policies Notes](https://docs.google.com/document/d/1OUwPCzzHa2YROSp7YugYG_LH-NNIiroweqVxxSrQ4gY/edit?usp=sharing)

#### Notes from meeting

- 

#### Task Items

- [ ]

**Comment by chelseybeck** [comment]
## 14 February
_[Back to Top](#)_

### AGENDA
_Timezones listed as PDT_

- [ ] 18:00 _5 minutes_ - Welcome!
   - [ ] Open Lead positions
      - [Co-lead](https://github.com/hackforla/ops/projects/1#column-19747877)
   - [ ] Intros
- [ ] 18:05 _10 minutes_ - Project support
- [ ] 18:15 _15 minutes_ - Review [board](https://github.com/hackforla/ops/projects/4) and assign new issues
- [ ] 18:30 _30 minutes_ - Terraform AWS IAM resources | working session
  - [ ] Decide what policies are needed for level 2 and beyond

### Resources
[IAM Groups & Policies Notes](https://docs.google.com/document/d/1OUwPCzzHa2YROSp7YugYG_LH-NNIiroweqVxxSrQ4gY/edit?usp=sharing)

#### Notes for the project support section of the meeting



#### Notes from meeting

- 

#### Task Items <br>

- [ ] Item 1

**Comment by ExperimentsInHonesty** [comment]
Add to next meeting agenda
- [ ] BW: created https://github.com/hackforla/ops/wiki/Permission-levels-for-Team-Members
- [ ] BW: revised issues
   - [ ] #111 
   - [ ] #109 
- [ ] BW: I am available to meet with the leads to help them with issue construction and board managment

**Comment by chelseybeck** [comment]
## 21 February
_[Back to Top](#)_

### AGENDA
_Times listed as PDT_

- [ ] 18:00 _10 minutes_ - Welcome!
   - [ ] Intros (if we have new members)
   - [ ] Recap from last meeting
- [ ] 18:10 _10 minutes_ - Project support (if you need more time, reach out in the #ops channel in Slack and we can adjust the agenda)
- [ ] 18:20 _5 minutes_ - [PR review](https://github.com/hackforla/devops-security/pulls)
- [ ] 18:25 _15 minutes_ - Review [board](https://github.com/hackforla/ops/projects/4) and assign new issues
- [ ] 18:40 _5 minutes_ - Discuss what policies are needed for level 2 and beyond
  - [ ] who needs greater than read access and why? 
- [ ] 18:45 _15 minutes_ - GitHub Action for Terraform Apply | working session

### Resources
[IAM Groups & Policies Notes](https://docs.google.com/document/d/1OUwPCzzHa2YROSp7YugYG_LH-NNIiroweqVxxSrQ4gY/edit?usp=sharing)

#### Notes for the project support section of the meeting
- Question for incubator team - are all resources currently tagged with a project identifier on creation? if not, is that a large lift?

#### Notes from meeting

- 

#### Task Items <br>

**Comment by chelseybeck** [comment]
## 13 March
_[Back to Top](#)_

### AGENDA
_Times listed as PDT_

- [x] 18:00 _10 minutes_ - Welcome!
   - [x] Intros (if we have new members)
   - [x] Recap from last meeting
- [x] 18:10 _10 minutes_ - Project support (if you need more time, reach out in the #ops channel in Slack and we can adjust the agenda)
- [x] 18:20 _5 minutes_ - [PR review](https://github.com/hackforla/devops-security/pulls)
- [x] 18:25 _15 minutes_ - Review [board](https://github.com/hackforla/ops/projects/4) and assign new issues
  - [ ] BW: created https://github.com/hackforla/ops/wiki/Permission-levels-for-Team-Members
  - [x] BW: revised issues
     - [x] #111 
     - [x] #109 
  - [ ] BW: I am available to meet with the leads to help them with issue construction and board management 
- [x] 18:40 _5 minutes_ - Discuss what policies are needed for level 2 and beyond
  - [ ] who needs greater than read access and why? 
- [x] 18:45 _15 minutes_ - Permissions testing | working session

### Resources
[IAM Groups & Policies Notes](https://docs.google.com/document/d/1OUwPCzzHa2YROSp7YugYG_LH-NNIiroweqVxxSrQ4gY/edit?usp=sharing)

#### Notes for the project support section of the meeting
- Question for incubator team - are all resources currently tagged with a project identifier on creation? if not, is that a large lift?

#### Notes from meeting

- 

#### Task Items <br>

**Comment by chelseybeck** [comment]
## 20 March
_[Back to Top](#)_

### AGENDA

- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Incubator - 15 min
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
- [ ] DevOps Security - 15 min
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/47#column-19771520) + Updates
- [ ] DevOps Project Board
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
### Resources
[IAM Groups & Policies Notes](https://docs.google.com/document/d/1OUwPCzzHa2YROSp7YugYG_LH-NNIiroweqVxxSrQ4gY/edit?usp=sharing)

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

- 

#### Task Items <br>

- [ ] Task

**Comment by chelseybeck** [comment]
## 27 March
_[Back to Top](#)_

### AGENDA

- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] DevOps Security - 15 min
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/47#column-19771520) + Updates
- [ ] Incubator - 15 min
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
- [ ] DevOps Project Board
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
### Resources
[IAM Groups & Policies Notes](https://docs.google.com/document/d/1OUwPCzzHa2YROSp7YugYG_LH-NNIiroweqVxxSrQ4gY/edit?usp=sharing)

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items <br>

- [ ] item 1

**Comment by jennyc910** [comment]
- @chelseybeck Could you please add the following to this week's agenda? https://github.com/hackforla/ops/issues/111

**Comment by kchotani** [comment]
@chelseybeck Could you please add the following to this week's agenda? 

Civic Tech Jobs is responsible for building out a [skills matrix](https://docs.google.com/spreadsheets/d/1_sCd8Kc2Tlhe--8UJLkgyDoqyQw_eZxhie3QTOwTHrM/edit#gid=1745353147) for volunteers at HfLA which will be used in future volunteer opportunities. We need help from DevOps in adding and classifying skills.

**Comment by chelseybeck** [comment]
## 27 March
_[Back to Top](#)_

### AGENDA

- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
- We propose that new items added to the agenda should have the initials of the person who added them, so that we can ask them followup questions if necessary.
   - BW: https://github.com/hackforla/ops/issues/111
   - BW: https://github.com/hackforla/ops/issues/91#issuecomment-2041725600
   - BW: https://github.com/hackforla/ops/wiki/Permission-levels-for-Team-Members needs an update
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/47#column-19771520) + Updates
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
- [ ] DevOps Project Board
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

Civic Tech Jobs is responsible for building out a [skills matrix](https://docs.google.com/spreadsheets/d/1_sCd8Kc2Tlhe--8UJLkgyDoqyQw_eZxhie3QTOwTHrM/edit#gid=1745353147) for volunteers at HfLA which will be used in future volunteer opportunities. We need help from DevOps in adding and classifying skills.

#### Notes from meeting

**Comment by drakeredwind01** [comment]
please add to next agenda:
- [ ] DRW: discussion of adding a discord server to hack for la https://discord.gg/M5AE4Vy4XPhttps://discord.gg/M5AE4Vy4XP

**Comment by chelseybeck** [comment]
## 17 April
_[Back to Top](#)_

### AGENDA

- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
  - [ ] DRW: adding a discord server to hack for la https://discord.gg/M5AE4Vy4XP
- [ ] DevOps Project Board
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/47#column-19771520) + Updates
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items

**Comment by ExperimentsInHonesty** [comment]
Notes from today's meeting: 
- issue needs to get made to develop requirements for discord
  - see issue #48 for example
  - https://stackoverflow.com/questions/59742017/how-to-add-or-remove-users-from-a-discord-server-via-the-api
  -

**Comment by ExperimentsInHonesty** [comment]
## 2024-04-24
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
- [ ] DevOps Project Board
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/47#column-19771520) + Updates
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by abbyz123** [comment]
Notes for today's meeting

- Docker issue, does devops team need to maintain our new docker our use docker shared with other team.  And update doc for the docker which can suit for M1 chip. 
- Help Drake to find a project which is suit for him
- Help Chelsey to build up the meeting agenda if no updates on Tuesday. 
- Fix Github issue #58 which can remind assignees to provide update weekly. 
- Devops security, establish AWS multi-factor authentication. (Github issue #22) assigned to Samuel Wang.
- Incubator, github #23 fix authentication error, credential cannot be loaded.

**Comment by freaky4wrld** [comment]
## 2024-05-09
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
- [ ] DevOps Project Board
  - [ ] [Questions](https://github.com/hackforla/ops/projects/4#column-19779585)
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [Questions](https://github.com/hackforla/devops-security/projects/1#column-19822962)
  - [ ] [New Issue Review](https://github.com/hackforla/devops-security/projects/1#column-19822959) + Updates
- [ ] Incubator
  - [ ] [Questions](https://github.com/hackforla/incubator/projects/1#column-19648744)
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by drakeredwind01** [comment]
https://github.com/hackforla/ops/issues/40#issuecomment-2101770502
COP lead needs to rewrite

**Comment by drakeredwind01** [comment]
https://github.com/hackforla/ops/issues/18#issuecomment-2101787245

**Comment by chelseybeck** [comment]
## 2024-05-15
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
  - [ ] Chelsey: branch protections are causing issues with terraform plan running github actions. Let's discuss forking vs cloning for project contributors
- [ ] DevOps Project Board
  - [ ] [Questions](https://github.com/hackforla/ops/projects/4#column-19779585)
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [Questions](https://github.com/hackforla/devops-security/projects/1#column-19822962)
  - [ ] [New Issue Review](https://github.com/hackforla/devops-security/projects/1#column-19822959) + Updates
- [ ] Incubator
  - [ ] [Questions](https://github.com/hackforla/incubator/projects/1#column-19648744)
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by chelseybeck** [comment]
action items: Chelsey to reach out to Gayle to find out more

**Comment by chelseybeck** [comment]
## 2024-05-22
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
- [ ] DevOps Project Board
  - [ ] [Questions](https://github.com/hackforla/ops/projects/4#column-19779585)
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [Questions](https://github.com/hackforla/devops-security/projects/1#column-19822962)
  - [ ] [New Issue Review](https://github.com/hackforla/devops-security/projects/1#column-19822959) + Updates
- [ ] Incubator
  - [ ] [Questions](https://github.com/hackforla/incubator/projects/1#column-19648744)
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by chelseybeck** [comment]
## 2024-05-29
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
- [ ] DevOps Project Board
  - [ ] [Questions](https://github.com/hackforla/ops/projects/4#column-19779585)
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
- [ ] DevOps Security 
  - [ ] PR Review
    - [ ] https://github.com/hackforla/devops-security/pull/34
  - [ ] [Questions](https://github.com/hackforla/devops-security/projects/1#column-19822962)
  - [ ] [New Issue Review](https://github.com/hackforla/devops-security/projects/1#column-19822959) + Updates
- [ ] Incubator
  - [ ] [Questions](https://github.com/hackforla/incubator/projects/1#column-19648744)
  - [ ] PR Review
    - [ ] https://github.com/hackforla/incubator/pull/73
    - [ ] https://github.com/hackforla/incubator/pull/71
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by chelseybeck** [comment]
## 2024-06-19
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
- [ ] DevOps Project Board
  - [ ] [Questions](https://github.com/hackforla/ops/projects/4#column-19779585)
  - [ ] [New Issue Review](https://github.com/hackforla/ops/projects/4#column-17567052) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [Questions](https://github.com/hackforla/devops-security/projects/1#column-19822962)
  - [ ] [New Issue Review](https://github.com/hackforla/devops-security/projects/1#column-19822959) + Updates
- [ ] Incubator
  - [ ] [Questions](https://github.com/hackforla/incubator/projects/1#column-19648744)
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by chelseybeck** [comment]
## 2024-06-26
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
- [ ] DevOps Project Board
  - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
  - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [Questions](https://github.com/hackforla/devops-security/projects/1#column-19822962)
  - [ ] [New Issue Review](https://github.com/hackforla/devops-security/projects/1#column-19822959) + Updates
- [ ] Incubator
  - [ ] [Questions](https://github.com/hackforla/incubator/projects/1#column-19648744)
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by chelseybeck** [comment]
## 2024-08-14
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
- [ ] DevOps Project Board
  - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
  - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
  - [ ] [Questions](https://github.com/hackforla/devops-security/projects/1#column-19822962)
  - [ ] [New Issue Review](https://github.com/hackforla/devops-security/projects/1#column-19822959) + Updates
- [ ] Incubator
  - [ ] [Questions](https://github.com/hackforla/incubator/projects/1#column-19648744)
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls)
  - [ ] [New Issue Review](https://github.com/hackforla/incubator/projects/1#column-19648741) + Updates
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by ExperimentsInHonesty** [comment]
Notes from todays meeting:

We have a problem with the domain homeunite.us where it's not resolving the http://dev.homeunite.us because of certbot is requiring that https://www.homeunite.us and https://homeunite.us be served by AWS.

**Comment by chelseybeck** [comment]
## 2024-08-21
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Project support
- [ ] Discussion
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by sudhara** [comment]
[Back to Top](https://github.com/hackforla/ops/issues/100#)

AGENDA
 Welcome!
 Intros (if new members)
 Project support
 Discussion

 Board Review
 [In Progress]  (https://github.com/orgs/hackforla/projects/73/views/1)

 DevOps Security
 [PR Review](https://github.com/hackforla/devops-security/pulls)

 Incubator
 [PR Review](https://github.com/hackforla/incubator/pulls)

 DevOps
 [PR Review](https://github.com/hackforla/ops/pulls)

Resources
Notes for the project support section of the meeting
If you are from a project and need DevOps support, please add notes here

Notes from meeting
Action Items (New Task Items)

**Comment by sudhara** [comment]
Aug 21 Minutes
- Create an issue for creating the Deployment Checklist (Sudha) 
   - Review the list of [Terraform Modules](https://github.com/hackforla/incubator/tree/main/terraform-modules/) to see if there are any other modules to be added to incubator for your project.
- Create an issue to auto-delete branch once PR is merged (Sudha)
- Create an issue for creating a glossary of terms. Please update Issue https://github.com/hackforla/incubator/issues/56 with the link (Bonnie)

**Comment by sudhara** [comment]
**AGENDA - 08-28-2024** 
 Welcome!
 Intros (if new members)
 Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D)

 Project support

 Discussion
- Review the pre-work checklist for new members

 Board Review
 [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
 [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates

 DevOps Security
 [PR Review](https://github.com/hackforla/devops-security/pulls)

 Incubator
 [PR Review](https://github.com/hackforla/incubator/pulls)

 DevOps
 [PR Review](https://github.com/hackforla/ops/pulls)

Resources
Notes for the project support section of the meeting
If you are from a project and need DevOps support, please add notes here

Notes from meeting
Action Items (New Task Items)

**Comment by sudhara** [comment]
Meeting minutes -

1. in the pre-work checklist creation backend.tfvars step needs to be moved down after the git pull - Alex - Done
2. status of different project AWS infrastructure setup status - create a template for posting status, issues etc - Sudha
3. Sudha to send a message to John Darragh to attend this meeting - Sep 11th. Create a new issue based on the spike "Create a new logging framework for applications to adopt" - Sudha

**Comment by sudhara** [comment]
## 2024-09-04
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
   - [ ] Review these repos with Chelsey (they are from before she took over Devops CoP) so that we can decide our responsibility/interaction.
    - [ ] https://github.com/100Automations/terraform-aws-terragrunt-modules
    - [ ] https://github.com/100Automations/terraform-loadbalanced-ecs-webapp
    - [ ] https://github.com/100Automations/terraform-aws-postgres-vpc
    - [ ] https://github.com/100Automations/terraform-aws-org-account 
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by sudhara** [comment]
## 2024-MM-DD
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
       - [ ] Review these repos with Leads (they are from before she took over Devops CoP) so that we can decide our responsibility/interaction.
            - [ ] https://github.com/100Automations/terraform-aws-terragrunt-modules
            - [ ] https://github.com/100Automations/terraform-loadbalanced-ecs-webapp
            - [ ] https://github.com/100Automations/terraform-aws-postgres-vpc
            - [ ] https://github.com/100Automations/terraform-aws-org-account
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by sudhara** [comment]
- Home unit us - incubator has only the DNS entry - is the hosted zone. dev, stage, prod sites are routed to EC2 instance (manual created resource). Terraform modules are created. Rough estimate 2-3 weeks to migrate the dev site. EC2 instance.  Incubator - currently a blocker that chelsey is working on.
- code changes to home unite us application github actions will be part of the respective repo.
- ecr resource needs to rewritten in incubator so that civic tech jobs application can use it or provide an alternate direction to create the container and how to create it in AWS. Create an issue for CoOp Leads to build this.  https://github.com/hackforla/incubator/tree/main/terraform-modules/multi-container-service

**Comment by sudhara** [comment]
## 2024-09-18
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
       - [ ] Review these repos with CoP Leads so that we can decide our responsibility/interaction.
            - [ ] https://github.com/100Automations/terraform-aws-terragrunt-modules
            - [ ] https://github.com/100Automations/terraform-loadbalanced-ecs-webapp
            - [ ] https://github.com/100Automations/terraform-aws-postgres-vpc
            - [ ] https://github.com/100Automations/terraform-aws-org-account
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by ExperimentsInHonesty** [comment]
Items for 2024-09-25 agenda
- [ ] Review the follow vaults and why we have them
   - [ ] devops-brittany (its empty)
   - [ ] devops-jimmy-pw (has an AWS login for JimmyJuarez10
- [ ] Review the users for the main ops vault and remove anyone no longer active on the team
   - [ ] Sophia Alice
      - [ ] keep
      - [ ] delete
   - [ ] Bonnie Wolfe
      - [x] keep
      - [ ] delete
   - [ ] Chelsey B
      - [x] keep
      - [ ] delete
   - [ ] Rabia Shaikh
      - [x] keep
      - [ ] delete
   - [ ] Nicole Dubin
      - [ ] keep
      - [ ] delete
   - [ ] Ernie
      - [ ] keep
      - [ ] delete
   - [ ] Zoey Zhang
      - [ ] keep
      - [ ] delete
   - [ ] Nayan Bhatt
      - [ ] keep
      - [ ] delete
   - [ ] Alex English
      - [x] keep
      - [ ] delete
   - [ ] Sudha Raamakrishnan
      - [x] keep
      - [ ] delete
   - [ ] Ryan Sakuma
      - [x] keep
      - [ ] delete

**Comment by ExperimentsInHonesty** [comment]
### Add to next agenda 
- The items for leads need to be moved to the leads agenda https://github.com/hackforla/ops/issues/118#issuecomment-2380148013

**Comment by sudhara** [comment]
## 2024-10-01
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by sudhara** [comment]
- Onboarding doc needs to be reviewed and updated. let's discuss in the leads meeting. preop checklist. Git, Terraform experience needed as a pre-req for new members. Slides needs to be prepared for the onboarding process. 
-

**Comment by sudhara** [comment]
- onboarding document needs to be created for ops team. 
- Tyler will create a PR for the leads to review to migrate HUU terraform code deployment.
- Issue to be created by Sudha for removing the peopledepot resources using the AWS console. Chelsey will identify the resources that needs to be removed.

**Comment by chelseybeck** [comment]
## 2024-10-16
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Discussion
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
- [ ] 6:30 PM Project support - please add notes below

### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by chelseybeck** [comment]
## 2024-10-23
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by chelseybeck** [comment]
## 2024-10-30
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by ale210** [comment]
## 2024-11-13
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by chelseybeck** [comment]
## 2024-11-20
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by ale210** [comment]
## 2024-11-27
_[Back to Top](#)_

### AGENDA
- [ ] Welcome!
  - [ ] Intros (if new members)
- [ ] Onboarding - [Next Steps](https://github.com/hackforla/devops-security/issues/new?assignees=&labels=Complexity%3A+Prework%2C+Feature%3A+Onboarding%2FContributing.md%2C+role+missing%2C+size%3A+1pt&projects=&template=pre-work-template-devops-security.md&title=Pre-work+Checklist%3A+DevOps-Security-Member%3A+%5Breplace+brackets+with+your+name%5D) 
- [ ] Project support
- [ ] Discussion
- [ ] Board Review
   - [ ] [Questions](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22Questions%2FReview%22)
   - [ ] [New Issue Review](https://github.com/orgs/hackforla/projects/73/views/1?filterQuery=status%3A%22New+Issue+Review%22) + Updates
- [ ] DevOps Security 
  - [ ] [PR Review](https://github.com/hackforla/devops-security/pulls)
- [ ] Incubator
  - [ ] [PR Review](https://github.com/hackforla/incubator/pulls) 
- [ ] DevOps
  - [ ] [PR Review](https://github.com/hackforla/ops/pulls)
### Resources

#### Notes for the project support section of the meeting
_If you are from a project and need DevOps support, please add notes here_

#### Notes from meeting

#### Action Items  (New Task Items)

**Comment by chelseybeck** [comment]
closing 2024 and opening 2025
- https://github.com/hackforla/ops/issues/148

**Comment by Rabia2219** [comment]
- https://github.com/hackforla/ops/issues/148
